src/component/main/Main.js

```diff
@@ -2,16 +2,29 @@ import React from 'react';
 
 import { Aside } from '../aside/Aside';
 import { Content } from '../content/Content';
+import { Greeting } from '../greeting/Greeting';
+import { Numbers } from '../numbers/Numbers';
 
-import './main.scss'
+import './main.scss';
 
-export const Main = () =>{
+export const Main = () => {
 
   return (
-   <div className='wrapper'>
-     <Aside />
-     <Content />
-   </div>
-  );
+    <div className='wrapper'>
+      <Aside/>
+      <Content/>
+      <Greeting/>
+      <Numbers
+        data = {
+          {
+            from: 5,
+            to:7
+          }
+        }
+        type='even'
```

Comment:
I think will be better to add this into data object property instead of setting a new props to the Numbers component

Reply:
changed the render logic, please take a look